python/seldon_core/batch_processor.py

```diff
@@ -376,7 +376,7 @@ def _send_batch_predict_multi_request(
         A string serialised result of the response (or equivalent data with error info)
     """
 
-    indexes = [x[0] for x in input_data]
+    indices = [x[0] for x in input_data]
```

Comment:
indexes is quite commonly used but we can change to indices

Reply:
changed back